### PR TITLE
Support TLS environment variables

### PR DESCRIPTION
**Pull request by mattcui**

### CODE REVIEW

File: updatecfg.go [hunk 1]
```diff
@@ -4,29 +4,45 @@ import (
...
+//	// DefaultTLSCert is the default value for HELM_TLS_CERT
+//	DefaultTLSCert = "$HELM_HOME/cert.pem"
+//	// DefaultTLSKeyFile is the default value for HELM_TLS_KEY_FILE
+//	DefaultTLSKeyFile = "$HELM_HOME/key.pem"
+//	// DefaultTLSEnable is the default value for HELM_TLS_ENABLE
+//	DefaultTLSEnable = false
+//	// DefaultTLSVerify is the default value for HELM_TLS_VERIFY
+//	DefaultTLSVerify = false
+//)
+
```
Comment (zhanggbj) [inline updatecfg.go:33]: If not used, please help to remove commented code, thx!

**Review by EmilyEmily — COMMENTED** [review]
install.sh needs to be updated:
PLUGIN_VERSION=${PLUGIN_VERSION:-"0.6.0"}

**Review by zhanggbj — APPROVED** [review]

**Comment by mattcui** [comment]
> install.sh needs to be updated:
> PLUGIN_VERSION=${PLUGIN_VERSION:-"0.6.0"}

@EmilyEmily I made a new install-binary.sh, please review.

File: install-binary.sh [hunk 2]
```diff
@@ -0,0 +1,135 @@
...
+fi
+
+# initArch discovers the architecture for this system.
+initArch() {
+  ARCH=$(uname -m)
+  case $ARCH in
+    armv5*) ARCH="armv5";;
+    armv6*) ARCH="armv6";;
+    armv7*) ARCH="armv7";;
+    aarch64) ARCH="arm64";;
```
Comment (EmilyEmily) [inline install-binary.sh:31]: Can these be in the else branch? Because we only support two ARCH: amd64 and i386.

File: install-binary.sh [hunk 3]
```diff
@@ -0,0 +1,135 @@
...
+# initOS discovers the operating system for this system.
+initOS() {
+  OS=$(uname | tr '[:upper:]' '[:lower:]')
+
+  case "${OS}" in
+    # Msys support
+    msys*) OS='windows';;
+    # Minimalist GNU for Windows
+    mingw*) OS='windows';;
+    darwin) OS='darwin';;
```
Comment (EmilyEmily) [inline install-binary.sh]: additional line

**Review by EmilyEmily — APPROVED** [review]

**Pull request merged**